src/azure-cli/azure/cli/command_modules/cosmosdb/aaz/latest/cosmosdb/postgres/private_endpoint_connection/__cmd_group.py

```diff
@@ -0,0 +1,23 @@
+# --------------------------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for license information.
+#
+# Code generated by aaz-dev-tools
+# --------------------------------------------------------------------------------------------
+
+# pylint: skip-file
+# flake8: noqa
+
+from azure.cli.core.aaz import *
+
+
+@register_command_group(
+    "cosmosdb postgres private-endpoint-connection",
```

Comment:
In network model, we have a centralized place to manage private_endpoint_connection and private_link_resource. For more details, please contact @necusjz for details.

Reply:
Removed private endpoint connection and private link related code, will open a new pr for them